plugins/lead-resources/src/components/CreateCustomer.svelte

```diff
@@ -190,7 +190,7 @@
         <EditableAvatar
           avatar={object.avatar}
           size={'large'}
-          name={object.name}
+          name={combineName(firstName?.trim() ?? '', lastName?.trim() ?? '')}
```

Comment:
It is not always true, since customer could be organization, so a check to extend a Person is required for this change.